app/views/navbar.scala.html

```diff
@@ -179,69 +179,64 @@
         });
     }
 
-    // Triggered when 'Sign In' in navbar is clicked
-    // Logs "Click_module=SignIn_location=Navbar_route=</|/audit|/help|...>"
-    $("#sign-in-button").on('click', function(){
+    // The rest of this script logs whichever button is hit in the navbar
+    // Example: Following logs "Click_module=PSLogo_location=Navbar_route=</|/audit|/help|...>"
+    $("#navbar-brand").on('click', function(){
+        logWebpageActivity("Click_module=PSLogo");
+    });
+
+    $("#navbar-sign-in-btn").on('click', function(){
         logWebpageActivity("Click_module=SignIn");
     });
 
-    // // Triggered when 'Start Exploring' in navbar is clicked
-    $(".navbar-start-btn").on('click', function(){
+    $("#navbar-start-btn").on('click', function(){
         logWebpageActivity("Click_module=StartExploring");
     });
 
-    // // Triggered when 'Start Validating' in navbar is clicked.
-    $(".navbar-validate-btn").on('click', function(){
+    $("#navbar-validate-btn").on('click', function(){
         logWebpageActivity("Click_module=StartValidating");
     });
 
-    // // Triggered when 'How to Label' in navbar is clicked.
-    $(".navbar-guide-btn").on('click', function() {
+    $("#navbar-guide-btn").on('click', function() {
         logWebpageActivity("Click_module=LabelingGuide");
     });
+    
+    $("#navbar-retake-tutorial-btn").on('click', function(){
+        logWebpageActivity("Click_module=RetakeTutorial");
+    });
+
+    $("#navbar-help-btn").on('click', function() {
+        logWebpageActivity("Click_module=Help");
+    });
 
-    // // Triggered when 'Results' in navbar is clicked.
-    $(".navbarResultsBtn ").on('click', function(){
-        logWebpageActivity("Click_module=Results");
+    // Following buttons are in the Data Drop Down Menu!!!
+    $("#navbar-results-btn").on('click', function() {
+        logWebpageActivity("Click_module=ResultsMap");
```

Comment:
Let's actually keep this as 'Results' instead of changing to 'ResultsMap' so that it is consistent with the old logs.